temporalio/worker/worker.py

```diff
@@ -250,6 +256,8 @@ def __init__(
                 data_converter=client_config["data_converter"],
                 interceptors=interceptors,
                 debug_mode=debug_mode,
+                disable_eager_activity_execution=disable_eager_activity_execution,
```

Comment:
nit: the server seems to call this `ActivityEagerExecution`, do we want to be consistent with the server? 

```
	// EnableActivityEagerExecution indicates if acitivty eager execution is enabled per namespace
	EnableActivityEagerExecution = "system.enableActivityEagerExecution"
```

https://github.com/temporalio/temporal/blob/master/common/dynamicconfig/constants.go#L91

Reply:
No, I don't think we want to be consistent with the server here. "eager activity execution" is clearer English than "activity eager execution" (adjective in front) and matches Go SDK (granted TS doesn't have a worker-level option and Java has `disableEagerExecution` which doesn't qualify that it's about activities, but I believe neither are as good as Go and Python).